src/algebra/geom_sum.lean

```diff
@@ -106,22 +138,34 @@ begin
   exact this
 end
 
-theorem geom_sum₂_mul_comm [ring α] {x y : α} (h : commute x y) (n : ℕ) :
+theorem commute.geom_sum₂_mul [ring α] {x y : α} (h : commute x y) (n : ℕ) :
```

Comment:
@bryangingechen, I'm not sure whether dot notation here is actually a good idea - it might make it harder to find `commute.geom_sum₂_mul` when all you can find is `geom_sum₂_mul`. I'm not sure how the vscode autocomplete behaves here.

Reply:
Right, and the fact that this conflicts with something in the root namespace could be bad. I do think this should be in the `commute` namespace. Do you have any ideas for a better name?

If we keep this name then we'd better make this theorem `protected`.

(The Lean server's autocomplete functionality is kind of unpredictable / unreliable. When it works it's great, but there are lots of oddities too; Kenny has posted a bunch of examples in Zulip about this before.)

Reply:
For now, @Julian-Kuelshammer, could you make all the declarations in the `commute` namespace in this file `protected`, if their names are the same as something in the root namespace?

Reply:
I think I did.